components/BillStatus/BillStatus.js

```diff
@@ -22,6 +25,26 @@ const BillStatus = (props) => {
             <div className="text-center">
               Bill Status
             </div>
+            <div className="text-center">
```

Comment:
What do you think about making a BillHistoryTable that we can reuse here and in BillHistory?